src/rules/variableNameRule.ts

```diff
@@ -92,8 +107,9 @@ function parseOptions(ruleArguments: string[]): Options {
         checkFormat: !banKeywords || hasOption(OPTION_CHECK_FORMAT),
         leadingUnderscore: hasOption(OPTION_LEADING_UNDERSCORE),
         trailingUnderscore: hasOption(OPTION_TRAILING_UNDERSCORE),
+        allCapsForConst: hasOption(OPTION_CONST_ONLY_FOR_CAPS),
```

Comment:
sorry, is it _all caps for const_ or _const only for caps_? please be consistent